proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -553,3 +553,11 @@ message ListQueuesResponse {
   repeated QueueInfo queues = 1;
   bytes next_page_token = 2;
 }
+
+message HealthCheckRequest {
```

Comment:
My main comment: please don't call this a "health check". We already have a thing called a health check, the standard grpc health check, with a health server, etc. Calling this (which is very different) a health check is very confusing.

Please rename everything to DeepHealthCheck or DependencyHealthCheck or something like that.

Reply:
Changed to deep health check